srcpkgs/deadbeef/patches/patch_skip_glade_translations.patch

```diff
@@ -0,0 +1,153 @@
+--- po/POTFILES.skip.orig	2021-02-20 21:51:50.867875091 -0800
++++ po/POTFILES.skip	2021-02-20 21:51:57.689896618 -0800
+@@ -4,3 +4,151 @@
+ plugins/pltbrowser/support.c
+ plugins/gtkui/support.c
++tools/glade/examples/editor/editor.glade
++tools/glade/examples/editor/src/interface.c
```

Comment:
Is this from upstream? Would be nice to have a reason for the patch.

Reply:
Yeah, all from upstream.

Reply:
Just put like `Patch from upstream`

Reply:
Oh, the patch was written by me, the example junk is from upstream.

Reply:
Commented